lib/plugins/create/templates/plugin/index.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+// https://github.com/serverless/serverless/blob/master/docs/04-extending-serverless/01-creating-plugins.md
+class ServerlessPlugin {
+  constructor(serverless, options) {
+    this.serverless = serverless;
+    this.options = options;
+
+    this.commands = {
+      deploy: {
+        lifecycleEvents: [
+          'functions',
+        ],
+        options: {
+          function: {
```

Comment:
how about changing it from function to `message`, so you can do `serverless hello --message "My message"` and it prints `Hello from Serverless: My message`